test/dummy/config/environments/development.rb

```diff
@@ -42,6 +44,12 @@
   # Print deprecation notices to the Rails logger.
   config.active_support.deprecation = :log
 
+  # Raise exceptions for disallowed deprecations.
+  config.active_support.disallowed_deprecation = :raise
```

Comment:
Nice 👍

Reply:
Oh that's actually just the default for the new feature for disallowed deprecations, and we don't have any disallowed deprecations.
But I now realize I haven't changed the comment above the change to raise exceptions for deprecations in the test environment. Will fix that.